lib/pure/ioselects/ioselectors_select.nim

```diff
@@ -279,8 +279,9 @@ proc updateHandle*[T](s: Selector[T], fd: SocketHandle,
         inc(s.count)
       pkey.events = events
 
-proc unregister*[T](s: Selector[T], fd: SocketHandle) =
+proc unregister*[T](s: Selector[T], fd: SocketHandle|int) =
```

Comment:
This should be ``IoHandle``. @dom96 is that a thing yet?

Reply:
nope. Pretty sure my upcoming PR brought these ``|int`` in (sorry).